packages/core/package.json

```diff
@@ -9,9 +9,9 @@
   "scripts": {
     "prepare": "run-s clean build",
     "build": "run-s build:js build:types",
-    "build:babel:cjs": "cross-env NODE_ENV=production BABEL_ENV=cjs babel ./src --ignore '**/*.stories.js,**/*.spec.js' --out-dir ./dist/cjs",
-    "build:babel:esm": "cross-env NODE_ENV=production BABEL_ENV=esm babel ./src --ignore '**/*.stories.js,**/*.spec.js' --out-dir ./dist/esm",
-    "build:babel:es": "cross-env NODE_ENV=production BABEL_ENV=es babel ./src --ignore '**/*.stories.js,**/*.spec.js' --out-dir ./dist/es",
+    "build:babel:cjs": "cross-env NODE_ENV=production BABEL_ENV=cjs babel ./src --out-dir ./dist/cjs --extensions \".js,.ts,.tsx\"",
```

Comment:
please don't remove the `--ignore` flags

Reply:
I moved it to the config so we could keep the scripts a bit shorter https://github.com/priceline/design-system/pull/871/files#diff-fc10a6682269c1fefc2abe05d1500b8aR31

Reply:
that works, I missed that. Just verify that they don't appear in `dist`